css-brochure-form/styles.css

```diff
@@ -0,0 +1,64 @@
+h2 {
+  font-family: 'Brandon Grotesque';
+  font-size: 1rem;
+  text-transform: uppercase;
+  text-align: center;
+  letter-spacing: 0.05rem;
+  color: rgb(235, 93, 86);
+}
+
+form {
+  font-family: 'Brandon Grotesque';
+  font-size: 0.7rem;
+  font-weight: 500;
+  text-transform: uppercase;
+  width: 40%;
+  margin: auto;
+}
+
+label {
+  padding-left: 10px;
+  line-height: 1;
+}
+
+form input {
+  margin-top: 3px;
+  margin-bottom: 15px;
+}
+
+form select {
+  margin-bottom: -10px;
```

Comment:
You definitely want to avoid negative values for margin and padding. If you are thinking about using a negative margin, it likely means that you just need to reduce the margin on a sibling element.